targets/switch/README.md

```diff
@@ -67,9 +67,9 @@ By default, DOCKER_IMAGE is set to 'bm-switchlink'.
 Invoking make without an explicit target builds the softswitch with only the
 auto-generated API (make bm).
 
-For details on the features supported by switchapi, switchsai, switchlink, and p4ofagent
-libraries, please refer to the README.md file in the switchapi, switchsai, switchlink,
-and p4ofagent repositories, respectively.
+For details on the features supported by switchapi, switchsai, and switchlink
```

Comment:
Looks like you are removing p4ofagent from the README file. Is this intentional?